scripts/aggregate-themes/theme-aggregator.js

```diff
@@ -33,127 +34,121 @@ class ThemeAggregator {
       // eslint-disable-next-line global-require, import/no-dynamic-require
       themeConfig = require(defaultConfig);
       const assets = ThemeAggregator.aggregateThemes({ ...themeConfig, ...theme && { theme } });
-      if (assets) {
-        return ThemeAggregator.writeJsFile(assets);
-      }
+      if (assets) return ThemeAggregator.writeJsFile(assets);
     }
+
     return null;
   }
 
   /**
-   * Aggregates theme assets.
-   * @param {string} theme - The theme to aggregate.
+   * Aggregates theme assets into a js file.
    * @param {Object} options - The aggregation options.
-   * @param {boolean} defaultFlag - Whether the theme to be generated is a root or scope theme. Guards against default theme and scope theme being equivalent.
-   * @returns {array} - An array of file names.
+   * @returns {array} - An array of aggregated theme files
    */
-  static aggregateTheme(themeName, options = {}, defaultFlag) {
-    const { theme, scoped = [] } = options;
+  static aggregateThemes(options) {
+    if (!ThemeAggregator.validate(options)) return null;
```

Comment:
Can we add braces around these ifs

Reply:
https://github.com/cerner/terra-toolkit/pull/376/commits/3ad5c5a087f95ed4e09c6caff52af29ad9a4a250

I am a little torn - I want to be succinct since these can be one-liners, but I understand the argument that brackets improve maintainability and extensibility. I am curious what others think?

Reply:
My personal preference is to always include brackets for if statements. I think they're easier to read and understand.